README.md

```diff
@@ -62,11 +62,8 @@ and options pytest-services adds:
 * `--aws-debug-calls` for printing (with `-s`) API calls we make
 * `--aws-profiles` for selecting one or more AWS profiles to fetch resources for or the AWS default profile / `AWS_PROFILE` environment variable
 * `--aws-regions` for selecting one or more AWS regions to fetch resources from or the default of all regions
-* `--aws-require-tag` adds Tag names to check on all EC2 instances for the `aws.ec2.test_ec2_instance_has_required_tags` test
-* `--aws-whitelisted-ports` adds ports to the whitelist for the `aws.ec2.test_ec2_security_group_opens_specific_ports_to_all` test
 * `--offline` a flag to tell HTTP clients to not make requests and return empty params
-* [`--severity-config`](#test-severity) path to a config file for marking different tests with different severities
-* [`--exemptions-config`](#test-exemptions) path to a config file for marking test name and resource IDs as expected failures
+* [`--config`](#custom-test-config) path to test custom config file
```

Comment:
nit: might be handy to link to the "Custom Test Config" from here

Reply:
as in the `config.yaml.example` file? Or the section in the README?

Reply:
I was thinking that that section of the readme

Reply:
Oh ok, yeah its doing that with the `(#custom-test-config)` bit, like it was with the `(#test-severity)` bit before. Do you mean having the link text be different? Or is what's there alright?

Reply:
d'oh missed that nm, what's there is good.